Dockerfile.acm

```diff
@@ -0,0 +1,49 @@
+FROM registry.redhat.io/ubi9/nodejs-20-minimal:1-51 AS web-builder
+
+USER 0
+
+WORKDIR /opt/app-root
+
+ENV HUSKY=0
+
+COPY Makefile Makefile
+COPY web/ web/
+
+# Replace the plugin name in the package.json for the custom build actions
+RUN sed -i 's/"name": "monitoring-plugin",/"name": "monitoring-console-plugin",/g' web/package.json
```

Comment:
should we add these under `/scripts`, WDYT?

Reply:
Makes sense to me, especially since it's duplicated in the deploy-acm.sh script